aries_cloudagent/config/tests/test_plugins_config.yaml

```diff
@@ -0,0 +1,7 @@
+plugins:
```

Comment:
I think plugins should be a map instead of a list. The key in the map would correspond to the plugin name. The name must be defined by the plugin. The value of the map is the "config" attribute, instead of being nested.